db/models/user.js

```diff
@@ -1,12 +1,12 @@
 const mongoose = require("mongoose");
+const Schema = require('mongoose').Schema
+const bcrypt = require('bcrypt');
 
-const Schema = mongoose.Schema;
-
-const UserSchema = new Schema({
```

Comment:
Why did you remove this?

Reply:
there should be required(bcrypt), resolve it

Reply:
I'm happy that this has been addressed